.github/actions/rust-unit-tests/action.yaml

```diff
@@ -45,8 +45,9 @@ runs:
     # Run the rust unit tests
     - name: Run all unit tests
       run: |
-        NEXTEST_EXPERIMENTAL_LIBTEST_JSON=1 cargo nextest run --profile ci --cargo-profile ci --locked --workspace --exclude smoke-test --exclude aptos-testcases --retries 3 --no-fail-fast --message-format libtest-json > nextest_output.json
+        NEXTEST_EXPERIMENTAL_LIBTEST_JSON=1 cargo nextest run --profile ci --cargo-profile ci --locked --workspace --exclude smoke-test --exclude aptos-testcases --retries 3 --no-fail-fast --message-format libtest-json > nextest_output.json || python3 .github/actions/rust-unit-tests/nextest_summary.py nextest_output.json "$GITHUB_STEP_SUMMARY" -f
```

Comment:
Hmm.. why has this been put here? Isn't there a separate PR for this? https://github.com/aptos-labs/aptos-core/pull/12767

Reply:
If the other doesn't merge first, I'll roll back this part in this PR. For now it's useful because there have been a lot of unrelated flaky tests.